pkg/reconciler/shared/tektonconfig/upgrade/upgrade.go

```diff
@@ -38,7 +38,8 @@ var (
 
 	// post upgrade functions
 	postUpgradeFunctions = []upgradeFunc{
-		upgradeStorageVersion, // upgrade #1: performs storage version migration
+		upgradeStorageVersion,              // upgrade #1: performs storage version migration
+		removeAllClusterTasksInstallerSets, // Remove all cluster tasks installersets
```

Comment:
Do I understand it correctly that after upgrading from 1.16 to 1.17, all clustertasks (even old versioned ones for 1.14 or 1.15) will be removed automatically?

Reply:
yes, all cluster tasks will be removed automatically